core-bundle/src/Doctrine/Schema/MysqlInnodbRowSizeCalculator.php

```diff
@@ -85,7 +85,10 @@ public function getInnodbRowSizeLimit(): int
     {
         static $size = null;
 
-        return $size ??= (int) $this->connection->executeQuery('SELECT @@innodb_page_size / 2 - 66')->fetchOne();
+        return $size ??= (int) (
+            $this->connection->executeQuery("SHOW STATUS LIKE 'innodb_page_size'")->fetchAssociative()['Value']
+            / 2 - 66
+        );
```

Comment:
```suggestion
        if (null === $size) {
            $res = $this->connection->executeQuery("SHOW STATUS LIKE 'innodb_page_size'");
            $size = (int) ($res->fetchAssociative()['Value'] / 2 - 66);
        }

        return $size;
```

Just looking for a nicer way to format this. 🙈